config/sample/host/01-ftc.yaml

```diff
@@ -692,3 +692,37 @@ spec:
     fields:
       - metadata.creationTimestamp
       - status
+---
+apiVersion: core.kubeadmiral.io/v1alpha1
```

Comment:
Should we include it by default? This means all users will start caching pods.

Reply:
Also would it cause problems with static pods?

Reply:
@gary-lgy probably not 🙂, i will move it to a separate file